src/containers/ProjectContentsContainer.jsx

```diff
@@ -9,6 +9,16 @@ import * as contentsActions from '../ducks/resource';
 import isElementTranslatable from '../helpers/isElementTranslatable';
 
 class ProjectContentsContainer extends Component {
+  static getTextFromPath(resource, path) {
```

Comment:
Don't feel very strong about it, but since we've got a `helpers` folder, perhaps this could go there.